lib/iex/lib/iex/autocomplete.ex

```diff
@@ -265,24 +265,37 @@ defmodule IEx.Autocomplete do
   end
 
   defp get_module_funs(mod) do
-    if function_exported?(mod, :__info__, 1) do
-      if docs = Code.get_docs(mod, :docs) do
-        Enum.filter_map(docs, &has_content?/1, &elem(&1, 0))
-      else
-        mod.__info__(:macros) ++ (mod.__info__(:functions) -- [__info__: 1])
-      end
+    docs = Code.get_docs(mod, :docs) || []
+    Enum.map(docs, &elem(&1, 0)) ++ module_info_funs(mod)
```

Comment:
Good call. What if we use the docs only if the module Kernel.SpecialForms?

-- 


*José Valimwww.plataformatec.com.br
<http://www.plataformatec.com.br/>Founder and Director of R&D*